src/main/java/bar/Simplest2.java

```diff
@@ -4,6 +4,9 @@
 
 	public String bar;
 	
-	public static void foo() {}
+	// TODO bam
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://rebloch.sonarsource.com/organizations/fab/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://rebloch.sonarsource.com/project/issues?id=test%3Afake-project-for-tests&pullRequest=1&issues=AWHcBRsyQYL-lH1H_cI-&open=AWHcBRsyQYL-lH1H_cI-)

[ISSUE_KEY]: AWHcBRsyQYL-lH1H_cI-